tools/pipeline_perf_test/test_suites/collector_batch_comparison/test-suite-docker.yaml

```diff
@@ -0,0 +1,326 @@
+# This test suite illustrates a basic setup for testing the openelemetry
+# go collector in 2 different configs - one with batching enabled and one
```

Comment:
nit: mention this is specifically comparing the effect of batching in Logs pipeline.